LiskHQ/lisk-explorer
# Get rid of config.docker.js file - Closes #504

### What was the problem?
Docker compose not working properly

### How did I fix it?
Added environment variables to config.js file

### How to test it?
Set the image to *lisk/explorer:2.0.0-alpha.1* on _docker-compose.yml_ file.
Run `docker-compose up`.
You should be able to access the application on *localhost:6040*

### Review checklist
- The PR solves #504
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
